tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -429,13 +429,14 @@ def file_renamed(self, prefix, test_name, real_name):
       os.rename(real_path, test_path)
 
   @contextmanager
-  def temporary_file_content(self, path, content):
+  def temporary_file_content(self, path, content, binary_mode=True):
```

Comment:
Keeping default as binary_mode, because the author originally had it set to `wb`.